src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -170,7 +171,7 @@ public FormValidation doCheckReturnStatus(@QueryParameter boolean returnStdout,
     /** If set to false, disables {@link Execution#watching} mode. */
     @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "public & mutable only for tests")
     @Restricted(NoExternalUse.class)
-    public static boolean USE_WATCHING = !"false".equals(System.getProperty(DurableTaskStep.class.getName() + ".USE_WATCHING"));
+    public static boolean USE_WATCHING = Boolean.parseBoolean(System.getProperty(DurableTaskStep.class.getName() + ".USE_WATCHING", Main.isUnitTest ? "true" : /* JENKINS-52165 turn back on by default */ "false"));
```

Comment:
It seems undesirable to not actually test using the mode that we are shipping to users, but I also see why it would be good to test against watch mode since any issues in downstream plugins are likely to be specific to that mode, so I am somewhat split on this.

Reply:
> It seems undesirable to not actually test using the mode that we are shipping to users

Indeed. On the other hand the non-watch mode has been tested for years, and I certainly hope this situation will be temporary.

Reply:
I'm okay with this part -- not perfect but it'll do.  Just trying to check that the exit condition stuff is 100% the same internally.